src/styles/themes/_default-theme.scss

```diff
@@ -790,7 +870,7 @@
     --button-secondary-disruptive-border-style
   );
   --button-secondary-disruptive-hover-border-color: var(
-    --button-secondary-disruptive-hover-background-color
+    --button-secondary-disruptive-border-color
```

Comment:
revert this, as these button colors are to spec.

Reply:
Got it, will do

Reply:
Reverted